src/imgix-core-js.js

```diff
@@ -117,6 +148,8 @@
     };
 
     ImgixClient.VERSION = VERSION;
+    ImgixClient.SHARD_STRATEGY_CRC = SHARD_STRATEGY_CRC;
```

Comment:
I don't really like exposing these constants--it makes things a little bit easier in our unit tests, but kind of muddies up the `ImgixClient` object. Can we remove these two lines and just redefine these constants somewhere in the test harness instead?

Reply:
The constants aren't just for testing. We expect library users to specify the shard strategy while instantiating `ImgixClient`.

Reply:
Alright, fair point.